pkg/model/model.go

```diff
@@ -1,13 +1,22 @@
 package model
 
 import (
+	"time"
+
 	"gorm.io/gorm"
 )
 
+type BaseModel struct {
+	ID        uint           `gorm:"primarykey" json:"id"`
+	CreatedAt time.Time      `json:"created_at"`
+	UpdatedAt time.Time      `json:"updated_at"`
+	DeletedAt gorm.DeletedAt `gorm:"index" json:"deleted_at"`
```

Comment:
I think it might be more convenient to use camelcase since the front end will use JavaScript? So that it fits with the rest of the JavaScript code conventions.